pypst/set.py

```diff
@@ -0,0 +1,33 @@
+from collections.abc import Mapping
+from dataclasses import dataclass
+from typing import Any
+
+from pypst.renderable import Renderable
+from pypst.utils import render_code
+
+
+@dataclass
+class Set:
```

Comment:
I'd prefer the name `SetRule` to better distinguish it from the `set` Python built-in.